src/constants/memory/hram.asm

```diff
@@ -48,20 +48,28 @@ hGameValuesSection::
 hNeedsUpdatingBGTiles::
  ds 1 ; FF90
 
-; bool value for enemie tile update
-; 0 = no update of enemie tiles needed,
-; 1 = update needed
-hNeedsUpdatingEnnemiesTiles::
+; Request the spriteslot at wC197 to be updated with the spritesheet
```

Comment:
@daid I'm still not sure about the content of `wC197`. Is it the offset of the tile to be copied during the current stage, or a spriteslot index?

Reply:
$C197 is the sprite slot index. So target vram address is `addr = ($8400 + $100 * [$C197])` and `[wLoadedEntitySpritesheets + [$C197]]` contains the number of which spritesheet to load.

Those numbers are composed of 3 bits which indicate the bank, and 5 bits indicate which 16 tile row to take from that bank. The upper 3 bits are the index into `NpcTilesBankTable` to find which bank to use.

Reply:
Ok, got it. I updated the naming and the comments, I think it should be correct now.